server/webhook.go

```diff
@@ -267,6 +268,67 @@ func (p *Plugin) postPullRequestEvent(event *github.PullRequestEvent) {
 	}
 }
 
+func (p *Plugin) handlePRDescriptionMentionNotification(event *github.PullRequestEvent) {
+	action := event.GetAction()
+	if action != "opened" {
+		return
+	}
+
+	body := event.GetPullRequest().GetBody()
+	if strings.Contains(body, "notifications@github.com") {
+		body = strings.Split(body, "\n\nOn")[0]
+	}
```

Comment:
Is it possible to open PRs using email?

Reply:
I'm not sure what you mean?

Reply:
That part of the code takes care of removing junk from comments that are created via email. Is it possible to create PRs from an email? If not, is this code piece needed?

Reply:
I see what you mean. No, as far as I'm aware you can't create PR's via email. Removed the if statement.